packages/create-cloudflare/e2e-tests/helpers.ts

```diff
@@ -0,0 +1,77 @@
+import { spawn } from "cross-spawn";
+
+export const keys = {
+	enter: "\x0d",
+	backspace: "\x7f",
+	escape: "\x1b",
+	up: "\x1b\x5b\x41",
+	down: "\x1b\x5b\x42",
+	right: "\x1b\x5b\x43",
+	left: "\x1b\x5b\x44",
+};
+
+export type PromptHandler = {
+	matcher: RegExp;
+	input: string[];
+};
+
+export type RunnerConfig = {
+	promptHandlers?: PromptHandler[];
+	argv?: string[];
+};
+
+export const runC3 = async ({
+	argv = [],
+	promptHandlers = [],
+}: RunnerConfig) => {
+	const proc = spawn("node", ["./dist/cli.js", ...argv]);
+	const output: string[] = [];
```

Comment:
nitpick, but why not calling this `stdout` to keep it consistent with `stderr`? 😅